src/main/kotlin/kotlinx/kover/api/KoverConfig.kt

```diff
@@ -328,10 +354,20 @@ public open class KoverMergedXmlConfig @Inject constructor(private val objects:
         config.execute(newClassFilter)
         classFilter.set(newClassFilter)
     }
+
+    /**
+     * Override annotation filter for the merged XML report generation task.
```

Comment:
I think documentation for 'override' action needs to be more specific here and in the example buildfile. It's not clear what 'override' does — does it remove previous filters? Or just adds new ones and select them in case of conflict?